### PR TITLE
Fix CI badges and cleanup rosdoc documentation

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
- Fix CI badges
- Update information on CI
- Remove unused rosdoc documentation

### CODE REVIEW

**Review by MarqRazz — APPROVED** [review]
@vatanaksoytezer I also found some errors on the getting started page. The second link should point to the tutorials repo

[This](https://github.com/vatanaksoytezer/moveit2_tutorials/blob/fix/cleanup-readme/doc/getting_started/getting_started.rst#create-a-colcon-workspace-and-download-moveit2-source) should be...

<pre>
mkdir -p ~/ws_moveit2/src
cd ~/ws_moveit2/src

wget https://raw.githubusercontent.com/ros-planning/moveit2/main/moveit2.repos
vcs import < moveit2.repos
<b>https://raw.githubusercontent.com/ros-planning/moveit2_tutorials/main/moveit2_tutorials.repos
vcs import < moveit2_tutorials.repos</b>
</pre>

**Comment by vatanaksoytezer** [comment]
Thanks for noticing. I am adding it to this PR since it's a small typo.

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ These tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/r
...
+- [![Build](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/industrial_ci_action.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/industrial_ci_action.yml?branch=main) [Tutorials + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
+- [![Format](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/format.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/format.yml?branch=main) [Format + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
+- [![Deploy](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/deploy.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/deploy.yml?branch=main) [Github Pages + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
```
Comment (tylerjw) [inline README.md:17]:
```suggestion
- [![Build](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/industrial_ci_action.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/industrial_ci_action.yml?query=branch%3Amain) [Tutorials + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
- [![Format](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/format.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/format.yml?query=branch%3Amain) [Format + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
- [![Deploy](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/deploy.yml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2_tutorials/actions/workflows/deploy.yml?query=branch%3Amain) [Github Pages + Github Actions](https://moveit2_tutorials.picknik.ai/): Latest (Foxy, MoveIt 2)
```

Here are some small changes I learned about recently that make the links actually go where you'd expect them.

Comment (vatanaksoytezer) [reply README.md:17]: Looks great, thanks for letting me know!

**Comment by MarqRazz** [comment]
Thanks for the help @tylerjw!

This looks good to merge @vatanaksoytezer.

**Pull request merged**